### PR TITLE
Class/method/field refactor

### PR DESCRIPTION
**Pull request by Nolan-Posey**
1. Refactored the Field, Method, and Class classes
2. Class, Field, and Method are all at 100% line code coverage
3. Removed Attribute and AttributeTests
4. Refactored MenuPrompts and CLI slightly to account for the changes to Field and Method
5. Refactored ClassAssets slightly to account for the removal of Attribute

### CODE REVIEW

File: src/Controller/MenuController.java [hunk 1]
```diff
@@ -251,36 +256,57 @@ public static void deleteAttribute(Class currentClass, Scanner scanner) {
...
+            //need to add message if no attributes exist
+            System.out.println(currentClass.displayAttributes());
+            System.out.print("\nchoose between 1 and " + (currentClass.getAttributes().size()) + " -> ");
+            try {
+                choice = Integer.parseInt(scanner.nextLine());
+            } catch (NumberFormatException e) {
+                System.out.println("Please enter a valid number");
+            }
+
+        } while (choice < 1 || choice > currentClass.getAttributes().size()+1);*/
```
Comment (TannerStephenson) [inline src/Controller/MenuController.java:277]: If there is no need for this, might just want to remove this to clean up the code.

File: src/Attributes/Field.java [hunk 2]
```diff
@@ -1,50 +1,84 @@
...
+        }
         this.primitive = primitive;
     }
 
+    /**
+     * Description: Gets the name of the field.
+     * @return - Returns a string with the name of the field.
+     */
+    public String getName() {
+        return name;
```
Comment (TannerStephenson) [inline src/Attributes/Field.java:47]: Might want to return this.name.

**Review by TannerStephenson — COMMENTED** [review]
Just some quick small comments

File: src/Controller/MenuController.java (hunk 1 above)
Comment (Nolan-Posey) [reply src/Controller/MenuController.java:277]: Removed the unnecessary comments

File: src/Attributes/Field.java (hunk 2 above)
Comment (Nolan-Posey) [reply src/Attributes/Field.java:47]: Good catch, I changed it to return this.name

**Pull request merged**